LLama/Sampling/DefaultSamplingPipeline.cs

```diff
@@ -25,15 +25,15 @@ public sealed class DefaultSamplingPipeline
     /// Number between -2.0 and 2.0. Positive values penalize new tokens based on their existing frequency in the text
     /// so far, decreasing the model's likelihood to repeat the same line verbatim.
     /// </summary>
-    public float AlphaFrequency
+    public float FrequencyPenalty
```

Comment:
Can we keep the old properties (as well), but tagged with an `[Obsolete]` attribute (including a message suggesting to use the new properties). That way we can leave that in for a couple of versions to steer people towards the new properties before making the breaking change,